lib/benchee/formatters/console.ex

```diff
@@ -62,27 +62,52 @@ defmodule Benchee.Formatters.Console do
     max_label_width + 1
   end
 
-  defp job_reports(jobs, label_width) do
-    Enum.map(jobs, fn(job) -> format_job job, label_width end)
+  defp job_reports(jobs, units, label_width) do
+    Enum.map(jobs, fn(job) -> format_job job, units, label_width end)
+  end
+
+  defp units(jobs) do
+    # Produces a map like
+    #   %{run_time: [12345, 15431, 13222], ips: [1, 2, 3]}
+    collected_values = jobs
+    |> Enum.flat_map(fn({_name, job}) -> Map.to_list(job) end)
+    # TODO: Simplify when dropping support for 1.2
+    # For compatibility with Elixir 1.2. In 1.3, the following group-reduce-map
+    # can b replaced by a single call to `group_by/3`
+    #   Enum.group_by(fn({measurement, _}) -> measurement end, fn({_, value}) -> value end)
```

Comment:
Could you please open a ticket for this? Like `Drop Elixir 1.2 support` and then we could have a check list what to change? Oh and mention that we'd probably do this with an Elixir 1.4 release :)